integration/lsp.test.ts

```diff
@@ -0,0 +1,127 @@
+interface LSPRequest {
+    method: string
+    params?: object[] | object
+}
+
+interface LSPResponse {
+    result?: string | number | boolean | object | null
+}
+
+function buildRequest(request: LSPRequest): string {
+    const json = JSON.stringify({...request, jsonrpc: "2.0", id: 1});
+    return `Content-Length: ${json.length}\r\n\r\n${json}`;
+}
+
+function parseResponse(data: string): LSPResponse {
+    const json: object = JSON.parse(data.split('\r\n')[2]);
+    return { result: json["result"] };
+}
+
+describe('LSP Server integration tests', () => {
+    let server;
+
+    beforeAll(async () => {
+        const { Server } = await import('@influxdata/flux-lsp-node');
+        server = new Server(false, false);
+    });
+
+
+    it('initializes', async () => {
+        const expected = {
+            "result": {
+                "capabilities": {
+                    "textDocumentSync":1,
+                    "referencesProvider":true,
+                    "definitionProvider":true,
+                    "renameProvider":true,
+                    "foldingRangeProvider":true,
+                    "documentSymbolProvider":true,
+                    "documentFormattingProvider":true,
+                    "completionProvider": {
+                        "resolveProvider":true,
+                        "triggerCharacters": [".",":","(",",","\""]
+                    },
+                    "signatureHelpProvider": {
+                        "triggerCharacters": ["("],
+                        "retriggerCharacters": ["("]
+                    },
+                    "hoverProvider":true
+                }
+            }
+        };
+
+        const request = {method: "initialize"};
+
+        const response = await server.process(buildRequest(request));
+        const error = response.get_error();
+        expect(error).toBe(undefined);
```

Comment:
I'm taking undefined here to basically mean there is no error expected.

Reply:
Yeah, that's what `.get_error` will spit out if there is no error.